dracut/30ignition/ignition-setup-base.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Ignition (setup base config)
+DefaultDependencies=false
+Before=ignition-complete.target
+
+Requires=local-fs-pre.target
```

Comment:
What's the point of this line?  It seems odd to have a `Requires=` here paired with `Before=`.

Reply:
Hmm good point. Think is what we do on CL, so it obviously doesn't hurt, and that target will get pulled in anyway. @bgilbert you added it in bootengine in https://github.com/coreos/bootengine/pull/135, was there a reason there?

Reply:
Looks like I copied it from `ignition-disks.service`, and the line originally dates back to https://github.com/coreos/coreos-overlay/commit/757e8014c2c0a9e35c2a445e76a343ac4414478d.

Reply:
OK, let's just remove the `Requires=` and get this merged.

Reply:
Fixed.